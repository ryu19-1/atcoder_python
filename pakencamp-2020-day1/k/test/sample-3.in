10
48 33 57 91 70 63 51 100 300 234
