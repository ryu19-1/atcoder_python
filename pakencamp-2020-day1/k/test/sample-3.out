1047
3
3
3
3
3
1
1
1
1
